src/components/tl/mlx5/tl_mlx5.c

```diff
@@ -28,6 +28,19 @@ static ucc_config_field_t ucc_tl_mlx5_lib_config_table[] = {
      ucc_offsetof(ucc_tl_mlx5_lib_config_t, dm_buf_num),
      UCC_CONFIG_TYPE_ULUNITS},
 
+    {"FORCE_REGULAR", "y",
+     "Force the regular case where the block dimensions "
```

Comment:
I would word this a bit differently, up to you.

"Enforce the regular case where the block dimensions evenly divide ppn. This option requires BLOCK_SIZE = 0."